app/design/frontend/base/default/template/algoliasearch/internals/configuration.phtml

```diff
@@ -139,9 +139,41 @@ if ($config->isInstantEnabled() && $isSearchPage) {
     </script>';
 }
 
+/** Generate API keys */
+$attributesToRetrieve = $config->getAttributesToRetrieve($customerGroupId);
+
+if ($attributesToRetrieve) {
+    $instantsearchApiKeyParams['attributesToRetrieve'] = $attributesToRetrieve;
+    $autocompleteApiKeyParams['attributesToRetrieve'] = $attributesToRetrieve;
+}
+
+/**
+ * Add custom instantsearch filter restrictions into $instantsearchApiKeyParams['filters'] variable
+ * Docs: https://www.algolia.com/doc/api-client/php/api-keys#generate-key
```

Comment:
I would put this doc link next to generateSearchSecuredApiKey call and put there the link to the filter param doc + explain 1 or 2 usecases